lib/msf/core/handler/reverse_http.rb

```diff
@@ -394,8 +394,8 @@ def on_request(cli, req)
             resp['Content-Type'] = 'application/octet-stream'
             resp.body = blob
 
-          rescue NoMethodError
-            print_error("Staging failed. This can occur when stageless listeners are used with staged payloads.")
+          rescue NoMethodError => e
+            print_error("#{e} Staging failed. This can occur when stageless listeners are used with staged payloads.")
```

Comment:
```suggestion
            print_error("#{e} Staging failed. This can occur when stageless listeners are used with staged payloads.")
            elog(e)
```